docs/source/getting_started/getting_started.md

```diff
@@ -423,7 +390,269 @@ curl -X POST "$URL/users/u1234/personalized_documents" \
+- `"snippet_id": { document_id: "effaffe",  sub_id: 2 }` containing `"Entscheidet sich der Auftraggeber nicht binnen eines Monats nach Zugang der Anzeige, so ist der\nAuftragnehmer  berechtigt,  den  Unterauftrag  entsprechend  dem  vorgelegten  Verhandlungsergebnis  abzuschließen.\n\nErteilt der Auftraggeber seine ausdrückliche Einwilligung zum Vertragsschluss oder erfolgt der Vertragsschluss nach\nAblauf der Monatsfrist, bleibt die Haftung des Auftragnehmers für die vertragsgemäße Ausführung seiner Leistungen\ngegenüber dem Auftraggeber unberührt."`
+
+## File Upload
+
+_The file upload features is not enabled by default_ contact Xayn if you do need it.
+
+Documents are often in different formats then plain text. When ingesting you can decide to provide a
+document "file" instead of a snippet. If you do so we will try to extract snippets from the given
+file.
+
+Currently this option can only be used together with the `split` option and follwing file formats are supported: `application/pdf`, `text/plain`, `text/html`.
+
+The "file" needs to be provided as base64 encoded string.
+Reusing the example from the getting started section and provding the snippet as `text/plain` file leads to following example:
+
+```bash
+curl -X POST "$URL/documents" \
+    --header "authorizationToken: $BACKOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "documents": [
+            {
+                "id": "xayn_cd5604c",
+                "file": "VGhlIHZvaWNlcyB0aGF0IGFyZSBkZW1hbmRpbmcgYmV0dGVyIHByaXZhY3kgcHJvdGVjdGlvbiBhbmQgb3duZXJzaGlwIG9mIG91ciBvd24gZGF0YSBhcmUgaW5jcmVhc2luZ2x5IGxvdWRlciwgdGhlcmUgaXMgYSBiYWNrbGFzaCB0b3dhcmRzIHRoZXNlIHByYWN0aWNlcy4gQXQgWGF5biwgb3VyIG1pc3Npb24gaXMgdG8gcHJvdmlkZSBwZXJzb25hbGlzYXRpb24gd2l0aG91dCB1c2VyIGRhdGEgbGVhdmluZyB0aGUgZGV2aWNlLCBtYWludGFpbmluZyBhYnNvbHV0ZSBwcml2YWN5LiBXZSB1c2Ugc2VtYW50aWMgc2ltaWxhcml0eSBhbmQgY2VudGVycyBvZiBpbnRlcmVzdCB0byB1bmRlcnN0YW5kIHVzZXIgcHJlZmVyZW5jZXMgYW5kIHByZXNlbnQgYmV0dGVyIG1hdGNoaW5nIGFydGljbGVzLiBXaXRoIG91ciBtb2RlbCBYYXluaWEsIHdlIG9mZmVyIHNlbWFudGljIHNpbWlsYXJpdHkgYW5kIHNlYXJjaCB3aXRoIG1pbmltYWwgZW5lcmd5IGNvbnN1bXB0aW9uIGFuZCBhdCBhIGxvdyBwcmljZSwgbWFraW5nIGl0IGhpZ2hseSBlbmVyZ3ktZWZmaWNpZW50IGNvbXBhcmVkIHRvIG90aGVyIHRyYW5zZm9ybWVyIG1vZGVscy4=",
+                "properties": {
+                    "title": "The initial challange",
+                    "link": "https://xayn.com/blog/the-initial-challenge",
+                    "image": "https://uploads-ssl.webflow.com/5ef08ebd35ddb63551189655/641320bc6be72c5453f4d98d_Blog%20Posts%20Visuals%20-%2003%20Mar%202023-p-2600.png",
+                    "location" : ["germany", "berlin", "conference"]
+                },
+                "split": true
+            }
+        ]
+    }'
+
+```
+
+Be aware that this always has the potential to produce [multiple snippets](#documents-with-multiple-snippets).
```

Comment:
```suggestion
```

I don´t think it adds much then what we already said.

Reply:
Especially given that we do remove the `split` from above I would say we should even highlight it more then it's now.

People often don't read documentation linearly and if you directly jump to file upload you might be very surprised by how it behaves. In general file upload might not be the best name. Something like text extraction might have been a better name.

Reply:
I removed the line here and rephrased the documentation above the example.